### ISSUE TITLE
Should truncation in an extremely long title that goes on for quite a few lines  be defined as an accessibility failure?

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#3724]

**Comment by mbgower** [comment]
This was just a test to see what happened in github and projects with long titles. I answered the question (which is that the issue is simply made multiple lines, not truncated).
